config/src/test/java/org/springframework/security/config/web/server/OAuth2LoginTests.java

```diff
@@ -176,6 +180,25 @@ public void defaultLoginPageWithSingleClientRegistrationThenRedirect() {
 		assertThat(driver.getCurrentUrl()).startsWith("https://github.com/login/oauth/authorize");
 	}
 
+	@Test
+	public void formLoginWithSingleClientRegistrationThenRedirect() {
+		this.spring.register(OAuth2LoginWithSingleClientRegistrations.class, OAuth2LoginWithFormLogin.class).autowire();
+		// @formatter:off
+		WebTestClient webTestClient = WebTestClientBuilder
+				.bindToWebFilters(new GitHubWebFilter(), this.springSecurity)
+				.build();
+		WebDriver driver = WebTestClientHtmlUnitDriverBuilder
+				.webTestClientSetup(webTestClient)
+				.build();
+		// @formatter:on
+		driver.get("http://localhost/");
```

Comment:
Please align the implementation to be similar to `defaultLoginPageWithMultipleClientRegistrationsThenLinks()`.

NOTE: You will need to add `DefaultLoginPage.assertLoginFormPresent()`